.github/workflows/tests.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm install -g yarn
+    - run: yarn && yarn add --frozen-lockfile node-rdkafka@2.7.4
+    - run: yarn lint
+    - name: Start Docker containers for Zookeeper, Kafka and Schema Registry
+      run: cd kafka-setup && docker-compose up -d
+    - run: yarn mocha --exit --timeout 60000 -R spec test/int/*
```

Comment:
can we use the script in the repo for this instead? https://github.com/nodefluent/kafka-streams/blob/master/package.json#L24 just to keep consistency npm/yarn, might be worth updating the package.json to use yarn instead of npm

Reply:
Yep. yarnified everything in separate typescript branch. Want to do this as separate line of work to have master build passing then create typescript branch and same should be happening. 

Mocha is quite bad for the async stuff - tests are not very stable. 
This begs to rewrite towards ava or something like it.